back/app/services/permissions/phase_permissions_service.rb

```diff
@@ -151,15 +151,16 @@ def volunteering_denied_reason_for_phase
     # Helper methods
 
     def posting_limit_reached?
-      if phase.posting_limited?
-        num_authored = phase.ideas.where(author: user, publication_status: 'published').size
-        return true if num_authored >= phase.posting_limited_max
-
-        if phase.allow_anonymous_participation?
-          author_hash = Idea.create_author_hash user.id, phase.project.id, true
-          num_authored_anonymously = phase.ideas.where(author_hash: author_hash).size
-          return true if (num_authored + num_authored_anonymously) >= phase.posting_limited_max
-        end
+      limit = phase.pmethod.posting_limit
+      return false unless limit
+
+      nb_posts = phase.ideas.where(author: user, publication_status: 'published').size
```

Comment:
personal preference possibly, but pref `num` to `nb` when shortening number

Reply:
> In English, nb is not a common abbreviation for "number." It's more often used in contexts where French influence is present, as "nb" stands for "nombre" (French for "number").

(ChatGPT circa 2020)